index.html

```diff
@@ -8,6 +8,36 @@
   </head>
   <body class="page">
     <div class="page__container">
+      <section class="advantages"> 
+        <div class="advantages__preview"><p class="advantages__heading">Устали от серости мегаполиса?</p><p class="advantages__heading-text">Нам есть, что вам<br> предложить!</p></div>
```

Comment:
@NazarPauk I'd suggest renaming the element from `advantages__preview` to `advantages__intro`. The term "preview" is typically used for images, small or preview content, something that precedes the full viewing of the content.

For text elements here, let's assign a single class, something like `advantages__intro-text`, and move all typography-related styles there, such as `font-size`, `font-weight`, `text-align`, and `line-height`. We'll also keep the resetting of default paragraph margins on `advantages__intro-text`, and for the differing margin on the second and subsequent paragraphs for mobile screens, we'll use the selector `.advantages__heading-text:not(:first-of-type)`.